types/min-app/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for min-app 1.0
+// Project: https://github.com/chobits4/min-app (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: My Self <https://github.com/chobits4>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
```

Comment:
- it's declaration files for Weapp(weixin) mini program
- So far the best preserved declaration files of typescript for mini program